### DISCUSSION TITLE
Feedback on Zen Browser Design Update

### DISCUSSION THREAD

**Discussion by abdullah-eksi** [zen-browser/desktop#3021] [Ideas]
In Zen browser’s latest update, the feature that displayed fixed elements as boxes within vertical tabs was removed. I believe the previous design was more visually appealing and organized. Displaying fixed elements as boxes conveyed their fixed nature more clearly, creating a more systematic look overall. It would be wonderful if users were given the option to switch back to the previous boxed design for fixed elements if they preferred. Thank you for all the effort you put into developing this browser; I truly appreciate your hard work.

**Comment by abdullah-eksi** [comment]
In Zen browser’s initial version, displaying fixed tabs in a boxed style made the interface look more organized and user-friendly. However, in the latest version (as shown in the image below), the fixed tabs now appear misaligned and somewhat cluttered, which detracts from the visual appeal. I believe that offering an option to revert to the boxed design, or fully restoring the previous layout, would result in a much cleaner and more elegant user experience.

## New Version 
![new_version](https://github.com/user-attachments/assets/6e7896ac-ef42-45ab-91e6-afc7c8f36ef3)

## Old Version
![old_version](https://github.com/user-attachments/assets/83bebcb8-5546-41a5-9f94-48f885b0c4bf)

**Reply by jannisborgers** [reply]
Just downloaded Zen for the first time, so I hope I understand correctly. 

You can still move tabs to the grid by right clicking and choosing "Add to essentials" instead of "Pin tab". These are two different ways to make tabs permanent.

**Reply by abdullah-eksi** [reply]
Thank you. This feature used to be called "Pin Tabs," but now they've moved it to the "Add to Essentials" section. It's exactly how I wanted it now. Thank you for your help.
